whittle/modules/linear.py

```diff
@@ -6,6 +6,8 @@
 
 
 class Linear(nn.Linear):
+    """An extension of PyTorch's `torch.nn.Linear` with support of sub-network dimensionality."""
```

Comment:
"""An extension of PyTorch's `torch.nn.Linear` with flexible input and output dimensionality corresponding to sub-network"""